client/src/components/profile/components/TimeLine.js

```diff
@@ -168,10 +144,38 @@ class Timeline extends Component {
   }
 }
 
+TimelineInner.propTypes = propTypes;
+
+function useIdToNameMap() {
```

Comment:
```suggestion
function createIdToNameMap() {
```
Nitpicky, I know, but `idToNameMap` isn't used by this function, it's created by it.

Reply:
It's a convention name for hooks

Reply:
https://reactjs.org/docs/hooks-custom.html

Reply:
Yeah, I've just been reading that.  I see why the convention is important now.  Thanks!